compiler/src/main/scala/dx/compiler/package.scala

```diff
@@ -0,0 +1,37 @@
+package dx.compiler
+
+import dx.core.ir.{Parameter, Value}
+import dx.translator.ParameterAttributes
+
+package object CompilerUtils {
+
+  // Return a parameter's dependencies on non-local files
+  def fileDependenciesFromParam(param: Parameter): Set[String] = {
```

Comment:
I would just put this as a protected method in the parent class `ExecutableCompiler`.

Reply:
Done.